content/tokio/tutorial/channels.md

```diff
@@ -175,11 +175,13 @@ async fn main() {
     let tx2 = tx.clone();
 
     tokio::spawn(async move {
-        tx.send("sending from first handle").await;
+        // send() returns a `Result` which cannot be ignored.
+        tx.send("sending from first handle").await.unwrap();
```

Comment:
Thanks! I don't think the comment is necessary for the documentation. It explains your change well, but I don't think someone who sees it with `unwrap()` from the beginning will be confused about it.

Reply:
Makes sense, I removed the comments.